.github/workflows/build_on_release.yml

```diff
@@ -41,14 +41,27 @@ jobs:
         mkdir -p commands
         go run ./cmd gendoc
 
+    - name: convert MD to HTML
+      uses: baileyjm02/markdown-to-pdf@v1
+      with:
+        input_dir: commands
+        output_dir: docs
+        # Default is true, can set to false to only get PDF files
+        build_html: true
+
+    - name: remove pdfs #This needs because action creates PDF by default
+      run: |
+        cd docs
+        sudo rm *.pdf
+
```

Comment:
What happens when we go ize.sh/docs?  I'd say we should rename ize.html to index.html, no?

Reply:
Renamed ize.html to index.html and rename links in html files